packages/server/src/__tests__/utils.test.ts

```
@@ -2,46 +2,52 @@
 mock.module('../api/system/environment', () => ({
   resolveEnvFile: jest.fn(() => '.env'),
 }));
 
+// Import after mocks are set up
+import { expandTildePath, resolvePgliteDir } from '../index';
+
 describe('Utility Functions', () => {
   beforeEach(() => {
-    mock.restore();
     // Reset environment variables
     delete process.env.PGLITE_DATA_DIR;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)